services/121-service/src/events/events.service.ts

```diff
@@ -60,6 +69,8 @@ export class EventsService {
     );
   }
 
+  // TODO: REFACTOR: put the parameters into an interface
+  // TODO: REFACTOR: why not call the input parameters oldRegistrationViews and newRegistrationsViews and always let them be arrays for simplicity?
```

Comment:
I was inspired here by the typeorm .save() method which also takes in an array or a single entity. Which I always thought was pretty convenient as the consumer of that package, but it does make the implementation of the log function a bit more complex.